Library/Homebrew/utils/shell.rb

```diff
@@ -67,12 +73,12 @@ def prepend_path_in_profile(path)
     end
 
     SHELL_PROFILE_MAP = {
-      bash: "~/.bash_profile",
+      bash: "~/.profile",
       csh:  "~/.cshrc",
       fish: "~/.config/fish/config.fish",
       ksh:  "~/.kshrc",
       mksh: "~/.kshrc",
-      sh:   "~/.bash_profile",
+      sh:   "~/.profile",
       tcsh: "~/.tcshrc",
       zsh:  "~/.zshrc",
```

Comment:
Should this be `.zprofile`?

Reply:
Maybe? But the safer bet is no. See https://superuser.com/a/187673/158392

Most people's login shell is `bash` (the typical default), and when they login, `~/.profile` will be sourced. Some people will then run `zsh` interactively, and when the do, `~/.zshrc` will be sourced, but `~/.zprofile` will not be sourced.

Some people will have `zsh` as their login shell, and when they login, both `~/.zprofile` and `~/.zshrc` will be sourced. So, `~/.zshrc` is the safer bet, because it gets sourced in both situations.

Reply:
Works for me 👍